packages/grid/_modules_/grid/locales/ptBR.ts

```diff
@@ -26,6 +26,11 @@ const ptBRGrid: Partial<GridLocaleText> = {
   toolbarFiltersTooltipShow: 'Exibir filtros',
   toolbarFiltersTooltipActive: (count) =>
     `${count} ${count !== 1 ? 'filtros' : 'filtro'} ${count !== 1 ? 'ativos' : 'ativo'}`,
+  
+  // Export selector toolbar button text
+  toolbarExport: 'Exportar',
+  toolbarExportLabel: 'Exportar',
+  toolbarExportCSV: 'Download como CSV',
```

Comment:
It's better to translate the entire phrase.

```suggestion
  toolbarExportCSV: 'Baixar como CSV',
```